src/NuGet.Core/NuGet.Common/Strings.resx

```diff
@@ -126,6 +126,10 @@
   <data name="Argument_Must_Be_GreaterThanOrEqualTo" xml:space="preserve">
     <value>Value must be greater than or equal to {0}</value>
   </data>
+  <data name="Error_DirectoryDoesNotExist" xml:space="preserve">
+    <value>Directory {0} does not exist.</value>
```

Comment:
Should we add single quotes, or similar, around the {0}, to better handle the long name, with potential spaces? What do we usually do?